packages/zowe-explorer/src/PersistentFilters.ts

```diff
@@ -37,13 +37,17 @@ export class PersistentFilters {
     private static readonly sessions: string = "sessions";
     private static readonly templates: string = "templates";
 
-    public schema: string;
+    public schema: PersistenceSchemaEnum;
     private mSearchHistory: string[] = [];
     private mFileHistory: string[] = [];
     private mSessions: string[] = [];
     private mDsTemplates: Types.DataSetAllocTemplate[] = [];
 
-    public constructor(schema: string, private maxSearchHistory = globals.MAX_SEARCH_HISTORY, private maxFileHistory = globals.MAX_FILE_HISTORY) {
+    public constructor(
+        schema: PersistenceSchemaEnum,
```

Comment:
We are free to change types on methods/constructors in the "zowe-explorer" package, right? I assumed this change was fine since it doesn't affect the ZE APIs. I can definitely add a changelog entry to the ZE API 🙂

Reply:
right 😋